src/main/resources/hazelcast-config-schema.json

```diff
@@ -1149,6 +1149,16 @@
       "required": [
         "class-name"
       ]
+    },
+    "LiteMember": {
+      "additionalProperties": false,
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "default": false,
+          "description": "When you want to use a Hazelcast member as a lite member, set this element's \"enabled\" attribute to true in that member's YAML configuration. Lite members do not store data, do not have partitions and are used mainly to execute tasks and register listeners."
```

Comment:
minor comment: if the description says "set this element's \"enabled\" attribute" then this description should be one nesting level higher (so the direct child of `"LiteMember"`).

Reply:
make sense. updated